src/routes/game/GameView.vue

```diff
@@ -768,8 +770,14 @@ export default {
     handleError(messageKey) {
       this.snackBarMessage = this.t(messageKey);
       this.showSnackbar = true;
+      this.snackBarColor = 'error';
       this.clearSelection();
     },
+    showCustomSnackbarMessage(messageKey) {
+      this.snackBarMessage = this.t(messageKey);
+      this.showSnackbar = true;
+      this.snackBarColor = 'surface-1';
+    },
```

Comment:
Will this trigger if the opponent makes the move? It looks like it's only called by the requests to make moves. I think we need to trigger it from a socket event instead in order to display it for both players as soon as it's resolved. 

Alternatively, we add a watcher on the top card which calls this func if the oldVal was not null but the new value is null. Then we wouldn't need to call it elsewhere